circle.yml

```diff
@@ -0,0 +1,10 @@
+machine:
+  environment:
+    AWS_REGION: 'ap-northeast-1'
+
+test:
+  post:
+    - mvn package
+    - mkdir $CIRCLE_ARTIFACTS/plugin
+    - mv /home/ubuntu/bitbucket-branch-source-plugin/target/*.hpi $CIRCLE_ARTIFACTS/plugin
+    - mv /home/ubuntu/bitbucket-branch-source-plugin/target/*.jar $CIRCLE_ARTIFACTS/plugin
```

Comment:
What's the point of this? Is the current [PR builder](https://jenkins.ci.cloudbees.com/job/plugins/job/bitbucket-branch-source-plugin/52/) not enough?